Sources/Stevenson/JiraService.swift

```diff
@@ -246,12 +195,17 @@ extension JiraService {
     public func setFixedVersion(_ version: Version, for ticket: String, on container: Container) throws -> Future<Response> {
         let fullURL = URL(string: "/rest/api/3/issue/\(ticket)", relativeTo: baseURL)!
 
-        return try container.client()
-            .put(fullURL, headers: self.headers) {
-                try $0.content.encode(VersionAddUpdate(version: version))
+        let logMessage = "Setting Fix Version field to <ID \(version.id ?? "nil")> (<\(version.name)>) for ticket <\(ticket)>"
+        self.logger.info("[JIRA] \(logMessage)")
+
+        return try container.make(SlowClient.self)
+            .put(fullURL, headers: self.headers, on: container) { request in
+                try request.content.encode(VersionAddUpdate(version: version))
+                self.logRequest(logMessage, request)
             }
             .catchError(.capture())
             .flatMap { response -> Future<Response> in
+                self.logResponse(logMessage, response)
```

Comment:
There's no equivalent to `observe(value:)`? `flatMap` ideally should not have any side-effects.

Reply:
Good point, fixed in 6b123d19a9d2b53a538600124a073035ffbd13ff